docs_src/microservices/application/Triggers.md

```diff
@@ -125,3 +125,109 @@ Designating an HTTP trigger will allow the pipeline to be triggered by a RESTful
 
 !!! note
     The data received, encoded as JSON or CBOR, must match the `TargetType` defined by your application service. The default  `TargetType` is an `Edgex Event`. See [TargetType](../AdvancedTopics/#target-type) for more details.
+
+## Custom Triggers
+
+It is also possible to define your own trigger and register a factory function for it with the SDK.  You can then configure the trigger by registering a factory function to build it along with a name to use in the config file.  These triggers can be registered with:
+
+```go
+sdk.RegisterCustomTriggerFactory("my-trigger-name", myFactoryFunc) 
+```
+
+!!! note
+    You can NOT override trigger names built into the SDK ("http", "messagebus", "edgex-messagebus", "external-mqtt") for a custom trigger.
```

Comment:
Remove "messagebus" as it will go away with the Ireland release. Code changes to remove it are TBD